### PR TITLE
chore(news): Add some fixes for improved waveforms blogpost

### PR DESCRIPTION
**Pull request by Holzhaus**
This fixes a few typos, adds proper monospace formatting for C++ classes/methods using backticks and adds a few links.

### CODE REVIEW

**Review by Swiftb0y — APPROVED** [review]
LGTM. but it's @m0dB blogpost so I'm waiting on their approval.

**Comment by m0dB** [comment]
thanks, Jan!

**Pull request merged**
